### DISCUSSION TITLE
Run a flash game that is split in several files

### DISCUSSION THREAD

**Discussion by oshustov** [ruffle-rs/ruffle#17223] [Q&A] [answered]
Hi! I need to run a game, that is split in several .swf files. Is it possible to run such game using Ruffle? I know where the entry point is, but when I'm trying to run it, it's broken, so I assume it doesn't see other files in the folder near the main file. If no, do you happen to know how to assemble those parts into a single file? Thanks.

<img width="586" alt="image" src="https://github.com/user-attachments/assets/e8638881-940b-462b-8a19-6af94fa76345">

**Comment by kjarosh** [comment, marked as answer]
Hi! Ruffle does support running games split across multiple SWFs, it should work out of the box. It is however possible that other .swf files are in the wrong place, or there are some other issues that prevent it from running.

You can:
1. open Ruffle logs and look for errors and warnings that suggest something's wrong,
2. try running the game using Flash Player and see whether that works.

If the game works in Flash Player and does not work in Ruffle, that's probably a bug in Ruffle and you can [file an issue](https://github.com/ruffle-rs/ruffle/issues).
